citadel.dme

```diff
@@ -1416,9 +1417,12 @@
 #include "code\game\machinery\teleporter\console.dm"
 #include "code\game\machinery\teleporter\pad.dm"
 #include "code\game\machinery\teleporter\projector.dm"
-#include "code\game\machinery\turrets\portable_turret.dm"
-#include "code\game\machinery\turrets\shielded_turret.dm"
```

Comment:
you aren't including the repathed shielded turret file

Reply:
.. or the renamed portable turret file

Reply:
fixed